scripts/installers/install_opengrm.sh

```diff
@@ -0,0 +1,10 @@
+# Miniconda3
```

Comment:
Please add some docs on what this script does.

Reply:
added doc lines